### ISSUE TITLE
I use command: pip install pymesh2,got an error

### ISSUE THREAD

**Issue by 760146355** [qnzhou/PyMesh#94]
Failed building wheel for pymesh2

**Comment by fmo91** [comment]
I'm facing the exact same problem. Have you found something?

**Comment by qnzhou** [comment]
Yeah, sorry, `pymesh2` on PyPI is out of date.  I will try to create a script to auto update it.

**Comment by 760146355** [comment]
I have tried your installation steps,but it is too complicated for me.
can you please fix this command：pip install pymesh2

**Comment by qnzhou** [comment]
For Linux and MacOS, I have created platform specific wheels: https://github.com/qnzhou/PyMesh/releases/tag/v0.2.0

It is possible to simply download the wheel file and `pip install [wheel_file]`.

The linux wheel, however, is not the `manylinux` version, which is required by PyPI.  Still, it is better than no linux release.  I would appreciate any help on getting `manylinux` wheel and Windows wheel support.

**Comment by lynetcha** [comment]
I was able to install pymesh2 using ``` pip install https://github.com/qnzhou/PyMesh/releases/download/v0.2.0/pymesh2-0.2.0-cp36-cp36m-linux_x86_64.whl
``` on my Ubuntu 14.04 machine but when I import pymesh, I get errors like ```The version GLIBCXX_3.4.22 not defined in file libstdc++.so.6```
I did not get such errors with the previous installation of pymesh through ```pip install pymesh2``` when it was still working.
One solution I have found is upgrading libstdc++6 on 16.04 machines, but I haven't been able to get this working on Ubuntu 14.04.

The full error on 14.04 is ```../lib/python3.6/site-packages/pymesh/lib/../third_party/lib/libcarve.so.2.0: symbol _ZTVNSt7__cxx1119basic_ostringstreamIcSt11char_traitsIcESaIcEEE, version GLIBCXX_3.4.21 not defined in file libstdc++.so.6 with link time reference```

**Comment by moncio** [comment]
Hi qnzhou, what's the way to install the library on Windows? Is possible with that converting an obj input to ply output? Thank's in advance

**Comment by ahmedmaalej** [comment]
This worked for me, I created virtual environment with setting python3 (if you want python2 you just replace 3 by 2) and followed only what is needed as system dependencies section in [here](https://pymesh.readthedocs.io/en/latest/installation.html). I choose to use pip (local) for dependencies installation rather than apt-get (global ) since I work under virtual environment, except the ones that cannot be installed with pip I did it with apt-get (or sudo apt-get):
` 
>virtualenv -p python3 PyMesh2 
 >cd PyMesh2
 >source bin/activate
 >python3 -m pip install --upgrade pip*
 >pip list
 >python3 -m pip install pymesh2
 >python3 -m pip install NumPy
 >python3 -m pip install SciPy
 >sudo apt-get install libeigen3-dev
 >python3 -m pip install TBB
 >sudo apt-get install libgmp-dev
 >sudo apt-get install libmpfr-dev
 >python3 -m pip install Boost
 >python3 -m pip install pymesh2 (error:Could NOT find SWIG (missing: SWIG_EXECUTABLE SWIG_DIR))

 swig -swiglib (check swig): 
 The program 'swig' can be found in the following packages:
 * swig
 * swig2.0
Try: sudo apt install 'selected package'

 >sudo apt-get install swig2.0
 > python3 -m pip install pymesh2

finally you will get:
Installing collected packages: pymesh2
Successfully installed pymesh2-0.1.14`

**Comment by AlbertoMQ** [comment]
Is there any progress on this?

**Comment by pmberkeley** [comment]
I am having trouble installing Docker (due to a Windows version I'm not able to update) and then the problem getting all the dependencies working in here. Also wondering if there's any progress on a Windows install.

**Comment by russell-sealand** [comment]
Now in late March 2021 'pip install pymesh2' is still not working in Windows - is there any progress on this? What is the workaround for Windows users? Thank you.

**Comment by Pheil** [comment]
The pip release page doesn't even list the latest version if it would install correctly.

**Comment by tintin10q** [comment]
I guess it just doesn't work on windows :(

**Comment by agamemnonc** [comment]
Would it be possible to have whl installers for Python 3.8 ? Thanks!

**Comment by Ieremie** [comment]
Any update on installing this on M1 macbooks?

**Comment by Ieremie** [comment]
> Any update on installing this on M1 macbooks?

After two days of banging my head, I managed to install it on my M1 MacBook Pro:

Follow the instructions from: https://pymesh.readthedocs.io/en/latest/installation.html

Install the third party dependencies using Macports, however make sure you do this using a terminal that is run with Rosseta. (`arch` must return i386 not arm64) - this is to ensure that the packages run as normal rather than complaining about the architecture.
In my case, for the 'boost' package, using MacPorts the instalation fails. Use `brew` instead, but again make sure you do this in a Rosseta terminal. I have installed `home-brew `with the Rosseta terminal too.
Follow the next steps in the instruction page to `build` and `install` pymesh: `./setup.py build` `./setup.py install`

**Comment by felixgodo** [comment]
Any update for Windows?
still "Could not build wheels for pymesh2, which is required to install pyproject.toml-based projects"

**Comment by mojoyup** [comment]
^^^SAME
